backend/src/test/java/com/google/sps/LoadUserPreferencesHelperTest.java

```diff
@@ -0,0 +1,106 @@
+package com.google.sps;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.MatchPreference;
+import com.google.sps.data.User;
+import com.google.sps.datastore.UserDatastore;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+
+@RunWith(JUnit4.class)
+public class LoadUserPreferencesHelperTest {
+  // Input constants
+  private static final String USERNAME_PERSON_A = "persona";
+  private static final int DURATION_DEFAULT = 30;
+  private static final String ROLE_DEFAULT = "Software engineer";
+  private static final String PRODUCT_AREA_DEFAULT = "Ads";
+  private static final List<String> INTERESTS_DEFAULT = Arrays.asList("Books");
+  private static final MatchPreference MATCH_PREFERENCE_DEFAULT = MatchPreference.ANY;
+
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+  @Mock private UserDatastore userDatastore;
+  @Mock private UsernameService usernameService;
```

Comment:
nit: the @Mock annotation is meant to be used with Mockito.initMocks(myTestClass). One can either initialize mocks manually by calling the mock method or automatically by using this annotation and calling initMocks on the test class. If initMocks sin't used the annotation does nothing.

I'm partial to manually calling mock and omitting the annotation because it ends up being cleaner in kotlin but you could go either way.